data/json/monster_special_attacks/spells.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "id": "pain_damage",
+    "type": "SPELL",
+    "name": "Pain",
+    "description": "Increases pain",
+    "valid_targets": [ "hostile" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -50,
+    "max_damage": -50,
+    "damage_increment": -50,
+    "effect": "recover_energy",
+    "effect_str": "PAIN"
+  },
+  {
+    "id": "stamina_damage",
+    "type": "SPELL",
+    "name": "Tired",
+    "description": "decreases stamina",
+    "valid_targets": [ "hostile" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -4000,
+    "max_damage": -4000,
+    "damage_increment": -4000,
+    "effect": "recover_energy",
+    "effect_str": "STAMINA"
+  },
+  {
+    "id": "mi-go_slaver_beam",
+    "type": "SPELL",
+    "name": "mi-go slaver beam",
+    "description": "Mi-go slaver beams cause pain, lower stamina, and give a short-lasting daze effect.  Starts very short range, increases rapidly with level.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "effect_str": "dazed",
+    "extra_effects": [ { "id": "pain_damage" }, { "id": "stamina_damage" } ],
+    "base_casting_time": 100,
+    "max_level": 3,
+    "min_range": 1,
+    "max_range": 5,
+    "range_increment": 5,
```

Comment:
i might make this 2 so you can have ranges in between 1 and 5. this spell also has a duration increment, but its max duration is the same as its min, so it will not do anything. having a value of 0 for increment is perfectly valid; leaving the increment field out entirely will default it to 0 as well.

Reply:
I actually want the range to jump up sharply just because of the mi-go that use it. Guards have a melee version of the spell, myrmidons have a close range version, and slavers use it at long range.